docs/Video Streaming for Navigation Apps/Video Streaming - Java/index.md

```diff
@@ -99,4 +111,77 @@ builder.setRPCNotificationListeners(onRPCNotificationListenerMap);
 ```
 
 ### Handling HMI Scaling (RPC v6.0+)
-If the HMI scales the video stream, you will have to handle scaling the projected view, touches and haptic rectangles yourself (this is all handled for you behind the scenes in the `VideoStreamManager`  API). To find out if the HMI scales the video stream, you must for query and check the `VideoStreamingCapability` for the `scale` property. Please check the [Adaptive Interface Capabilities](Displaying a User Interface/Adaptive Interface Capabilities) section for more information on how to query for this property using the system capability manager.  
\ No newline at end of file
+If the HMI scales the video stream, you will have to handle scaling the projected view, touches and haptic rectangles yourself (this is all handled for you behind the scenes in the `VideoStreamManager` API). To find out if the HMI scales the video stream, you must for query and check the `VideoStreamingCapability` for the `scale` property. Please check the [Adaptive Interface Capabilities](Displaying a User Interface/Adaptive Interface Capabilities) section for more information on how to query for this property using the system capability manager.
```

Comment:
I wonder if "Handling HMI Scaling" should be moved to the bottom of the guide since this paragraph is only relevant if the developer isn't using the Video Stream Manager.